src/download.test.js

```diff
@@ -1,6 +1,12 @@
+import axios from "axios";
+import MockAdapter from "axios-mock-adapter";
+
 import { SkynetClient, defaultSkynetPortalUrl } from "./index";
 
+const mock = new MockAdapter(axios);
+
 const portalUrl = defaultSkynetPortalUrl;
+const hnsLink = "doesn";
```

Comment:
let's use foo for more general test usage

```suggestion
const hnsLink = "foo";
```